homeassistant/components/command_line/manifest.json

```diff
@@ -2,5 +2,6 @@
   "domain": "command_line",
   "name": "Command Line",
   "documentation": "https://www.home-assistant.io/integrations/command_line",
-  "codeowners": []
+  "codeowners": [],
+  "quality_scale": "internal"
```

Comment:
Is this internal? As in, this is an integration one is configuring, right? There are no deps that bring them in?

Reply:
```suggestion
```